modules/backend/lang/en/lang.php

```diff
@@ -141,7 +141,7 @@
             'name_field' => 'Name',
             'description_field' => 'Description',
             'is_new_user_default_field_label' => 'Default group',
-            'is_new_user_default_field_comment' => 'Add new administrators to this group by default',
+            'is_new_user_default_field_comment' => 'Add new administrators to this group by default.',
```

Comment:
That would be a separate issue, we'd have to audit everywhere date formats are used within October, assign them keys based on commonalities and then reference those keys to change the datetime formats associated with them. I imagine something like that would be integrated as a backend setting. 

I do agree with the premise though, I would also like more control over how October displays dates by default across the system. I know you can change the options for specific instances but it would be convenient to be able to modify the defaults system wide.